rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -134,18 +143,18 @@ public String login(String login, String pass, File cred, String ssh) throws Res
                 entity = new MultipartEntity();
                 entity.addPart("credential", new FileBody(cred, "application/octet-stream"));
             }
-            method.setEntity(entity);
+            httpPost.setEntity(entity);
 
-            HttpResponse response = httpClient.execute(method);
+            HttpResponse response = httpClient.execute(httpPost);
             String responseAsString = convertToString(response.getEntity().getContent());
             switch (response.getStatusLine().getStatusCode()) {
                 case 200:
                     break;
                 default:
                     String message = responseAsString;
```

Comment:
it doesn't look like we need a new String message. We can just use responseAsString variable.